### DISCUSSION TITLE
Hatch Rest+ 2nd Gen

### DISCUSSION THREAD

**Discussion by yuejon** [dahlb/ha_hatch#16] [Ideas]
Just found this integration and it works brilliantly with my Hatch Rest Mini. However, I also have a Hatch Rest+ 2nd Gen and it does not seem to be supported yet. Any plans to include that soon?

Thanks!

**Comment by dahlb** [comment]
I've used it great with my rest plus since 1.7.0, not sure what you're experiencing, feel free to open an issue with details and logs
